terraform/20-app/cognito.tf

```diff
@@ -48,8 +48,14 @@ module "cognito" {
   user_pool_name    = "${local.prefix}-user-pool"
   client_name       = "${local.prefix}-client"
   user_pool_domain  = "${local.prefix}-domain"
-  callback_urls     = ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk/api/auth/callback/cognito"]
-  logout_urls       = ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk"]
+  callback_urls = concat(
+    ["https://${terraform.workspace}.dev.ukhsa-dashboard.data.gov.uk/api/auth/callback/cognito"],
```

Comment:
We'll need to revise the structure of these URLs at some point so we can account for all non-dev environments. But I think this is fine for now